addons/point_of_sale/static/src/app/utils/money_details_popup/money_details_popup.xml

```diff
@@ -12,9 +12,9 @@
                             <div class="money-details-value d-flex align-items-center w-100 my-1 mx-auto" t-on-input="updateMoneyDetailsAmount">
                                 <input class="pos-input form-control w-50 text-end" t-att-id="moneyValue" type="number" t-model.number="state.moneyDetails[moneyValue]" t-on-focus="ev=>ev.target.select()"/>
                                 <label class="oe_link_icon w-25 text-end" t-att-for="moneyValue">
-                                    <t t-if="currency.position === 'before'"><t t-esc="currency.symbol"/>&nbsp;</t>
+                                    <t t-if="currency.position === 'before'"><t t-esc="currency.symbol"/>&#160;</t>
```

Comment:
why don't you remove `&nbsp;` altogether? You can use the `mx-1` bootstrap class if you want to add some padding. This way you will also be able to only have only one `t` tag: `<t t-if="..." t-esc="..." />`

Reply:
Just a note: NBSP means non-breaking space. The goal is to keep the symbol and the amount together in one line. If you can find a css fix for that, better.